bin/v-copy-user-package

```diff
@@ -38,6 +38,10 @@ if [ ! -z $1 ]; then
     if [ ! -z $2 ]; then
         # Copy package
         cp -f $HESTIA/data/packages/$package.pkg $HESTIA/data/packages/$new_package.pkg
+        # Don't leave the .sh file behind
+        if [ ! -f $HESTIA/data/packages/$package.sh ]; then 
+            cp $HESTIA/data/packages/$package.sh $HESTIA/data/packages/$new_package.sh
```

Comment:
```suggestion
            mv $HESTIA/data/packages/$package.sh $HESTIA/data/packages/$new_package.sh
```
When `cp` is used copy of `.sh` file is leaved behind.

Reply:
copy makes a copy of the package and does not rename it see line 40 
cp -f $HESTIA/data/packages/$package.pkg $HESTIA/data/packages/$new_package.pkg

Script creates an 1 to 1 copy of the package ..